assets/model_monitoring/components/model_monitor/model_monitor_output_signal_metrics/spec.yaml

```diff
@@ -0,0 +1,52 @@
+$schema: http://azureml/sdk-2-0/SparkComponent.json
+type: spark
+
+name: model_monitor_output_signal_metrics
+display_name: Model Monitor - Output Signal Metrics
+description: Output the computed model monitor metrics to the default datastore.
```

Comment:
can you remove the default datastore here?

Reply:
yes, removed.